Templates/_JsonTemplate.json

```diff
@@ -690,7 +690,7 @@
                 "EnableVersioning": true
             },
             "Fields": [
```

Comment:
Remove 692-694. Title overriding never worked as mentioned in issue

Reply:
I suggest removing those lines aswell @okms